src/main/java/lotto/exception/ExceptionMessage.java

```diff
@@ -0,0 +1,29 @@
+package lotto.exception;
+
+public enum ExceptionMessage {
+    ERROR("[ERROR] "),
```

Comment:
여러가지 예외의 가능성들을 `enum`으로 잘 추출하신 것 같아요!!
네이밍도 너무 좋습니다! 👍 

그런데, `ERROR`의 부분은 예외의 종류가 아닌, 메시지 포맷과 관련된 상수라고 생각해요.
이 친구는 `enum` 객체로 두는 것 보다 내부에서 상수로 추출하는 것이 더 자연스럽지 않을까 생각합니다!
```suggestion
    private static final String PREFIX = "[ERROR] "
```

Reply:
Enum 객체와 상수 간의 관계를 고려하지 못했던 것 같습니다.. 좋은 충고 감사합니다